submit_and_compare/static/html/submit_and_compare_edit.html

```diff
@@ -18,7 +18,15 @@
           </div>
           <span class="tip setting-help">This assigns an integer value representing the weight of this problem</span>
         </li>
-        
+
+        <li class="field comp-setting-entry is-set">
+          <div class="wrapper-comp-setting">
+                <label class="label setting-label" for="submit_and_compare_edit_max_attempts">Max Attempts</label>
+                <input class="input setting-input" id="submit_and_compare_edit_max_attempts" step="1" min="1" value="{{ max_attempts }}" type="number">
```

Comment:
Am I correct in thinking a zero signifies unlimited attempts? If so, why is the "min" 1?